rich/_windows.py

```diff
@@ -18,7 +19,10 @@ class WindowsConsoleFeatures:
     from ctypes import wintypes
     from ctypes import LibraryLoader
 
-    windll = LibraryLoader(ctypes.WinDLL)  # type: ignore
+    if sys.platform == "win32":
```

Comment:
What was this solving

I suspect that if windll is None, it will break on L38.

Reply:
Reverting to old method with a type ignore